src/lib.rs

```diff
@@ -145,7 +147,7 @@
 //! | Feature name  | Description | Available PubNub APIs |
 //! | :------------ | :---------- | :------------- |
 //! | `full`        | Enables all non-conflicting features | Configuration, Publish, Subscribe, Access Manager, Parse Token, Presence |
-//! | `default`     | Enables default features: `publish`, `subscribe`, `serde`, `reqwest`, `aescbc`, `std` | Configuration, Publish, Subscribe |
+//! | `default`     | Enables default features: `publish`, `subscribe`, `serde`, `reqwest`, `crypto`, `std` | Configuration, Publish, Subscribe |
```

Comment:
add `crypto` into `full` feature.

btw. do we want to have it as a default one?

Reply:
@Xavrax, I will add, however, regarding the issue of `default`, I do not possess a clear opinion. I would suspect that we enable everything by default, or no?

Reply:
I'm not sure if we should enable everything as default. 
Imo we should have as small amount default features as possible but with taking to the account dx of our customers.